README.md

```diff
@@ -13,13 +13,13 @@ For more information, please visit [https://support.cockroachlabs.com](https://s
 Install the package:
```

Comment:
@fantapop should L6/L7 be updated here? (Github won't let me comment on those lines as they're outside the diff...)

Reply:
hmm, yeah I wonder why that's not updated.  That file is generated from this template.  Here is the line:

https://github.com/cockroachdb/cockroach-cloud-sdk-go/blob/main/internal/spec/templates/README.mustache#L7

Reply:
It looks like this needs to be updated: https://github.com/cockroachdb/cockroach-cloud-sdk-go/blob/e2fef1f9062095e3e71531634ea65145e3943557/internal/spec/config.yaml#L2